app/src/main/res/values/strings.xml

```diff
@@ -32,6 +32,7 @@
   <string name="cached_content">You\'re currently offline. Here\'s some cached content.</string>
   <string name="offline_alert">You\'re currently offline.</string>
   <string name="welcome">Welcome.</string>
+  <string name="drawer_welcome">Welcome</string>
```

Comment:
The fix to the lint issue if to upload all the translations to crowdin and download again. 
But I don't want to upload this because it is really similar to the other `welcome`. you just don't have the dot at the end. 

That's why I will do it and push to your branch soon.